pyproject.toml

```diff
@@ -11,7 +11,7 @@ skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 branch = true
 
 [tool.coverage.report]
-fail_under = 92
+fail_under = 90
```

Comment:
A reduction? I think our standards say that we shouldn't go down, do you think that is applicable here?

Reply:
Reverted